hazelcast/src/main/java/com/hazelcast/util/IterableUtil.java

```diff
@@ -71,4 +72,46 @@ public void remove() {
     public static <T> Iterable<T> nullToEmpty(Iterable<T> iterable) {
         return iterable == null ? Collections.<T>emptyList() : iterable;
     }
+
+    public static <T> Iterable<T> createIterableWithSingleItem(final T item) {
```

Comment:
I also do not see actual usage of this particular method:
- it isn't being used in this PR
- there is no test for it.